### PR TITLE
Implement settings section

### LINKED ISSUE #5: Implement settings section
### Context

The admin panel requires a settings section, accessible to the owner. In this section, we are going to show a list of settings that can be updated in the smart contract. For instance, the amount of tokens to be burned to obtain an NFT is a configurable parameter in the smart contract. Then, we will have a line in this settings page where the number can be changed. When the number is changed, we need a confirm button, that will dispatch the smart contract transaction to update the value. This step will require Metamask, or the connected wallet, confirmation the execute the operation.

### Acceptance Criteria

- The settings section is implemented
- The settings section includes all the configurable parameters from the smart contract
- We are able to update parameters in testnet

### PR DESCRIPTION
**Pull request by viganzeqiri**
Closes #5

### CODE REVIEW

File: src/sections/settings/token-settings/TokenSettings.jsx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                  <Iconify icon="charm:tick" />
+                </IconButton>
+              </Tooltip>
+            </Grid>
+          </>
+        ) : (
+          <>
+            <Grid item sm={10}>
+              <Typography variant="subtitle1">
+                Tokens to burn for receiving an NFT
```
Comment (xn3cr0nx) [inline src/sections/settings/token-settings/TokenSettings.jsx:161]: This should be named Burn Ratio, will be know to admin what it means

**Review by xn3cr0nx — CHANGES_REQUESTED** [review]
Just a request change atm, also I'm still able to login with my personal user that is not included into the multi sig, can you verify it?

File: src/sections/settings/token-settings/TokenSettings.jsx (hunk 1 above)
Comment (viganzeqiri) [reply src/sections/settings/token-settings/TokenSettings.jsx]: Added a fix for that :)

**Review by xn3cr0nx — APPROVED** [review]
LGTM

**Pull request merged**